src/app/components/client/ExposureCardDataClass.tsx

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ReactElement } from "react";
+import styles from "./ExposureCard.module.scss";
+import { SubscriberBreach } from "../../../utils/subscriberBreaches";
+import { useL10n } from "../../hooks/l10n";
+import { ScanResult } from "../../functions/server/onerep";
+
+type DataBrokerDataClassProps = {
+  scanResultData: ScanResult;
+  exposureCategoryLabel: string;
+  num: number;
+  icon: ReactElement;
+};
+
+export const DataBrokerDataClass = (props: DataBrokerDataClassProps) => {
+  const emailsList = (
+    <ul className={styles.emailsList}>
+      {props.scanResultData.emails.map((email: string, index: number) => (
+        <li key={index}>{email}</li>
+      ))}
+    </ul>
```

Comment:
Since this HTML is (as far as I can see) exactly the same between breaches and data brokers, you could even consider moving that to `<ExposureDataClassLayout>`, and just passing it `props.scanResultData.emails` / `props.subscriberBreachData.emailsEffected`.